resources/puppetlabs/lein-ezbake/template/pe/ext/default.erb

```diff
@@ -13,7 +13,14 @@ USER="<%= EZBake::Config[:user] %>"
 GROUP="<%= EZBake::Config[:group] %>"
 INSTALL_DIR="/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>"
 CONFIG="/etc/puppetlabs/<%= EZBake::Config[:real_name] %>/conf.d"
+
+# Bootstrap path
+<% if EZBake::Config[:bootstrap_source] == 'services-d' -%>
+BOOTSTRAP_CONFIG="/etc/puppetlabs/<%= EZBake::Config[:real_name] %>/services.d/,${INSTALL_DIR}/config/services.d/"
```

Comment:
There's two paths in there with a comma, the second one is under `/opt/puppetlabs/server/apps/{project}/config`. I included `config/`just to make it clear, since like you've pointed out, `/opt` isn't where you usually see these kinds of files. I think there's a desire to potentially move some of the integral `conf.d` files that users shouldn't be editing under /opt as well